images/openshift-golang-builder.yml

```diff
@@ -7,6 +7,15 @@ content:
       branch:
         target: rhel-9-golang-1.20
       url: git@github.com:openshift/ocp-build-data.git
+    modifications:
+    - action: replace
+      match: "# FOD wrapper modification"
+      replacement: "COPY go_wrapper.sh /tmp/go_wrapper.sh
```

Comment:
I think I'd rather see this script in Dockerfile directly, than the comment to be used as a replacement placeholder.

Reply:
ok, seem directly hack in dockerfile is straightforward